### PR TITLE
Extend security snoozes

### PR DESCRIPTION
**Pull request by n1zyy**
### Description
Snooze the three CVEs that `rake security` is complaining about

### Acceptance Criteria
- [ ] You agree these are not urgent security flaws.

### CODE REVIEW

File: .security.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 CVES:
 # placeholder to make non-nil array
   CVE-no-such-number: 2020-01-01
-  CVE-2020-11023: 2021-10-31
-  CVE-2021-41098: 2021-10-31
-  CVE-2021-41136: 2021-10-31
+  CVE-2020-11023: 2021-11-30
```
Comment (n1zyy) [inline .security.yml:4]: jQuery: https://nvd.nist.gov/vuln/detail/CVE-2020-11023

File: .security.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+  CVE-2021-41098: 2021-11-30
```
Comment (n1zyy) [inline .security.yml:5]: nokogiri vulnerability on jruby, which we don't use: https://nvd.nist.gov/vuln/detail/CVE-2021-41098

File: .security.yml [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
+  CVE-2021-41136: 2021-11-30
```
Comment (n1zyy) [inline .security.yml:6]: puma vulnerability behind a reverse proxy (Apache Traffic Server) which we don't use: https://nvd.nist.gov/vuln/detail/CVE-2021-41136

**Comment by jcq** [comment]
Closing due to this PR being incorporated into #16905

**Pull request closed without merging**